bench/test_hpack.py

```diff
@@ -0,0 +1,33 @@
+import hpack.hpack
+
+
+class TestHpackEncodingIntegersBenchmarks:
+    def test_encode_small_integer_large_prefix(self, benchmark):
+        benchmark(hpack.hpack.encode_integer, 120, 7)
+
+    def test_encode_small_integer_small_prefix(self, benchmark):
+        benchmark(hpack.hpack.encode_integer, 120, 1)
```

Comment:
Minor: I think it might be worth adding keyword arguments here, i.e.

```python
benchmark(encode_integer, integer=120, prefix_bits=1)
```

to make it clear that these are arguments for `encode_integer()`, not `benchmark()`.

(Obvious in hindsight, but confused me the first time I looked at this.